lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -129,6 +131,9 @@ class LineChartBarData {
   /// to show dot spots upon the line chart
   final FlDotData dotData;
 
+  // to show line chart annotations such as average line and vertical dot lines
+  final ExtraLinesData extraLinesData;
```

Comment:
Here we don't need it, because ExtraLinesData as defined on LineChartData, we should use that to draw lines in the whole chart, and not related any LineChartBarData,
Please remove it, and for the below lines as I said we should put it on the BelowBarData, I will explain it on the next sections.

Reply:
Remove it but put it where..?

Reply:
As I saw you defined it to the LineChartData

Reply:
Removed.